src/jmcomic/jm_plugin.py

```
@@ -1306,3 +1309,25 @@ def update_failed_count(self, client: AbstractJmClient, domain: str):
     def failed_count(client: JmcomicClient, domain: str) -> int:
         # noinspection PyUnresolvedReferences
         return client.domain_req_failed_counter.get(domain, 0)
+
+
+class DownloadCoverPlugin(JmOptionPlugin):
+    plugin_key = 'download_cover'
+
+    def invoke(self,
+               dir_rule: dict,
+               size='',
+               photo: JmPhotoDetail = None,
```

Ruff (0.13.1)

`JmPhotoDetail` may be undefined, or defined from star imports

(F405)